docs/dev/style-guide.rst

```diff
@@ -39,17 +39,25 @@ Content
 * Do not break the content across multiple lines at 80 characters,
   but rather break them on semantic meaning (e.g. periods or commas).
   Read more about this `here <https://rhodesmill.org/brandon/2012/one-sentence-per-line/>`_.
-* If you are cross-referencing to a different page within our website,
+* If you are :ref:`cross-referencing <style-guide:Cross-references>` to a different page within our website,
   use the ``doc`` role and not a hyperlink.
-* If you are cross-referencing to a section within our website,
+* If you are :ref:`cross-referencing <style-guide:Cross-references>` to a section within our website,
   use the ``ref`` role with the label from the `autosectionlabel extension <http://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html>`__.
+* Use ``<abstract concept>`` and ``<variable>`` as placeholders in code and URLs. For instance:
+
+  * ``https://<slug>.readthedocs.io``
+  * ``:guilabel:`<your username>` dropdown``
+* Make sure that **all bullet list items end with a period**, and don't mix periods with no periods.
+
 
 Word List
 ---------
 
 We have a specific way that we write common words:
 
 * ``Git repository`` for the place that stores Git repos. We used to use ``VCS``, but this is deprecated.
+* ``Git provider`` for generic references to GitHub/Bitbucket/GitLab/Gitea etc.
+  We avoid "host" and "platform" because they are slightly more ambiguous.
```

Comment:
Better suggestions are most welcome :pray:

Reply:
Yea, we also call them integrations in some places too I think? It would be good to be explicit when we're talking about Git-specific hosts that are actually used for code & auth, so I think Git provider is fine? Though provider is a bit abstract as well, tbh, but so are host & platform 🤷

Reply:
As you also noted in the PR adding texts to the Dashboard UI of "Connected Services", we would want to keep changes to the current Dashboard minimal. In the upcoming Dashboard frontend, we will _hopefully_ recall our terminology and find it natural to add some more tweaks both in the Dashboard and in the documentation. The future version of the documentation will also be less complicated to impose changes to - I feel somewhat confident about that :crossed_fingers: